content/docs/start/data-and-model-access.md

```diff
@@ -82,7 +82,7 @@ bring in changes from the data source later, using `dvc update`.
 > doesn't actually contain a `get-started/data.xml` file. Like `dvc get`,
 > `dvc import` downloads from [remote storage](/doc/command-reference/remote).
 
-`.dvc` files created by `dvc import` have special fields, such as the data
+`.dvc` files created by `dvc import` have special fields — such as the data
 source `repo`, and `path` (under `deps`):
```

Comment:
Hmm, I don't think dash works here.

Reply:
It's either a dash or nothing. A colon would be excessive and the comma splice is wrong. Brackets would look odd since there'd be nested parentheses.

Reply:
Long dashes are usually for jumping ahead in the story, which isn't the case here. Instead, here we want the reader to notice certain fields in the snippet below. https://www.dummies.com/education/language-arts/grammar/how-to-use-long-and-short-dashes-in-your-writing/ (BTW we may not be using this right everywhere ATM.)

Comma is very common before "such as" https://www.grammarly.com/blog/such-as-comma/ (again for nonrestrictive clauses, such as this one)

Reply:
argh same MW-inspired point :)

The problem with this sentence is that `(),:` are all already present.

Reply:
On this one I do have a not-so-weak opinion I must say. I see `blah blah — bleh:` ending in `:` and think something is wrong. No problem in having 2 commas in a sentence, I think?

Reply:
btw just looked at your link (this time dummies.com rather than MW) and once again think it agrees with me. Actually I agree completely with it: "the curator painted the dinosaur orange — everyone hates the color" is wrong and "the curator painted the dinosaur orange — a color hated by everyone" is correct. This is the same case.

I would not use a comma as a pause just before a comma-separated list as that would confuse things. "... have special fields, such as X, and Y" could be interpreted as "have 1. special fields (such as X), and 2. Y."

A dramatic pause (--) is absolutely essential here to show that "such as" applies to both X and Y. Otherwise in most cases I agree comma before "such as" is fine.

Reply:
> I would not use a comma as a pause

OK you should use your own style of course. But this text is already published. What problem are we fixing?

> fields, such as X, and Y" could be interpreted ...

I see. In this case I think the improvement would be to remove the comma between "`repo`, and `path`"

Reply:
done